pkg/tinydtls/doc.txt

```diff
@@ -33,4 +33,21 @@
  * If neither `DTLS_PSK` nor `DTLS_ECC` are specified, then tinyDTLS will
  * enable `DTLS_PSK` by default.
  *
+ * Configuration options
+ * ---------------------
```

Comment:
Actually I was thinking about keeping the grouping as you had it in the header. It should be possible here.

Reply:
I'm not sure I understand what you mean. Can you point me to any example?

Reply:
Not really but I was thinking about something like this:

```diff
diff --git a/pkg/tinydtls/doc.txt b/pkg/tinydtls/doc.txt
index bd2fc69177..3e96f8b278 100644
--- a/pkg/tinydtls/doc.txt
+++ b/pkg/tinydtls/doc.txt
@@ -32,22 +32,47 @@
  *
  * If neither `DTLS_PSK` nor `DTLS_ECC` are specified, then tinyDTLS will
  * enable `DTLS_PSK` by default.
+ */
+
+/**
+ * @defgroup    tinydtls_config Tinydtls configuration header
+ * @ingroup     pkg_tinydtls config
+ * @brief       Provides compile-time configuration for tinydtls
  *
- * Configuration options
- * ---------------------
- *
- * TinyDTLS uses the following options with a default value. The value can be
- * changed at compile time using `CFLAGS`.
- *
- * - `DTLS_CONTEXT_MAX`     - The maximum number of DTLS context at the same time.
- *                          Default value is 2.
- * - `DTLS_PEER_MAX`        - The maximum number DTLS peers (i.e. sessions)
- *                          Default value is 1.
- * - `DTLS_HANDSHAKE_MAX`   - The maximum number of concurrent DTLS handshakes.
- *                          Default value is 2.
- * - `DTLS_SECURITY_MAX`    - The maximum number of concurrently used cipher keys.
- *                          Default value is `DTLS_HANDSHAKE_MAX + DTLS_PEER_MAX`.
- * - `DTLS_HASH_MAX`        - The maximum number of hash functions that can be
- *                          used in parallel.
- *                          Default value is `3 * DTLS_PEER_MAX`.
+ * @{
+ */
+/**
+ * @brief The maximum number of DTLS context at the same time
+ */
+#ifndef DTLS_CONTEXT_MAX
+#define DTLS_CONTEXT_MAX    (2)
+#endif
+
+ /**
+ * @brief The maximum number DTLS peers (i.e. sessions)
+ */
+#ifndef DTLS_PEER_MAX
+#define DTLS_PEER_MAX       (1)
+#endif
+
+ /**
+ * @brief The maximum number of concurrent DTLS handshakes
+ */
+#ifndef DTLS_HANDSHAKE_MAX
+#define DTLS_HANDSHAKE_MAX  (2)
+#endif
+
+ /**
+ * @brief The maximum number of concurrently used cipher keys
+ */
+#ifndef DTLS_SECURITY_MAX
+#define DTLS_SECURITY_MAX   (DTLS_HANDSHAKE_MAX + DTLS_PEER_MAX)
+#endif
+
+ /**
+ * @brief The maximum number of hash functions that can be used in parallel
  */
+#ifndef DTLS_HASH_MAX
+#define DTLS_HASH_MAX       (3 * DTLS_PEER_MAX)
+#endif
+/** @} */
```

Reply:
I see. Never knew that is possible. Changed it. Also removed added lines in `Makefile.include` as that is not needed anymore.